lib/improver/tests/helper_functions_ensemble_calibration.py

```diff
@@ -71,6 +71,36 @@ def set_up_temperature_cube():
     return set_up_cube(data, "air_temperature", "K")
 
 
+def set_up_spot_cube(data, phenomenon_standard_name, phenomenon_units):
```

Comment:
So this is the only place we would have to update the code if there was a new site-specific cube format? Would we just need to make sure there is a new unit test to cover it?